mica-webapp/src/main/resources/_templates/studies.ftl

```diff
@@ -5,12 +5,18 @@
 <!-- Template variables -->
 <#if !type??>
   <#assign title = "studies">
+  <#assign className = "Study,HarmonizationStudy">
```

Comment:
Redundant with the "type".

Reply:
This is because it will be used with `Mica_study.className` such that it will be reflected in the widget in the search page.